include/oneapi/tbb/task_arena.h

```diff
@@ -295,6 +298,13 @@ class task_arena : public task_arena_base {
         r1::execute(*this, func);
         return func.consume_result();
     }
+
+    d2::task_group_status wait_for_impl(d2::task_group& tg) {
+        d2::task_group_status status;
```

Comment:
```suggestion
        d2::task_group_status status = d2::task_group_status::not_complete;
```
In the implementation of `isolated_task_group` there is also an assertion that the resulting status is not `not_complete`. Not sure if it's useful, given the implementation of `task_group::wait`; on the other hand, it certainly describes the expected invariant.

Reply:
It'll definitely won't hurt :)
Added the assertion.